### DISCUSSION TITLE
How do I access Raspap when on holiday

### DISCUSSION THREAD

**Discussion by faustXman** [RaspAP/raspap-webgui#1298] [Q&A]
How can I access my raspap dashboard when I am outside home network, in a new location?
I tried 10.3.141.1 but it doesn't work.

**Comment by netsolv** [comment]
Need some more details... are you trying to access raspap console on a device still connected to your home network and you're away? Is the raspap device with you and you;re connected to the raspap SSID and the console is not connecting? Are you trying to use DNS to access the dashboard? Any additional information would be helpfu...

**Reply by faustXman** [reply]
I think I've sort it up. I had to change to uap before the first reboot.
